lib/irida/pipelines.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'open-uri'
+require 'uri'
+require 'net/http'
+require 'irida/pipeline'
+
+module Irida
+  # Module that reads a workflow config file and registers the available pipelines
+  module Pipelines
+    mattr_accessor :register_pipelines, :find_pipeline_by
+    cattr_accessor :available_pipelines, :pipeline_config_dir, :pipeline_schema_file_dir
+
+    @@pipeline_config_dir = 'config/pipelines' # rubocop:disable Style/ClassVars
```

Comment:
**nitpick (blocking):** can you change these `@@variable` to normal UPPER_CASE_VARIABLE except for `@@available_pipelines = []` which should be changed to `@available_pipelines = []`

Reply:
Updated variables to get rid of the rubocop errors caused by the use of @@ ind08fd2d05586261bfb12aafc47e7e971a61f711f